### DISCUSSION TITLE
Make Lenses menu scrollable

### DISCUSSION THREAD

**Discussion by webard** [laravel/nova-issues#6310] [Ideas & Feature Requests] [answered]
Hello,

I have some Resources with many Lenses. Can Lenses menu be scrollable like filters menu?

<img width="184" alt="image" src="https://github.com/laravel/nova-issues/assets/855788/49ca69d3-31a1-4cfb-89af-c601a99b7cf4">

Thank you

**Comment by crynobone** [comment, marked as answer]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
